src/Backend/Core/Engine/Base/AjaxAction.php

```diff
@@ -41,9 +38,9 @@ public function execute()
      *
      * @return Response
      */
-    public function getContent()
+    public function getContent(): Response
     {
-        $statusCode = (isset($this->content['code']) ? $this->content['code'] : 200);
+        $statusCode = $this->content['code'] ?? self::OK;
```

Comment:
Well, my argument is not that we shouldn't use it because it is new. My argument is that I don't find the code readable, and I think it adds more complexity to understand it

@forkcms/core-contributors any opinions?